docs/README.md

```diff
@@ -1,34 +1,69 @@
 
-Triggers enables users to map fields from an event payload into resource
-templates. Put another way, this allows events to both model and instantiate
-themselves as Kubernetes resources. In the case of `tektoncd/pipeline`, this
-makes it easy to encapsulate configuration into `PipelineRun`s and
-`PipelineResource`s.
+# Overview of Tekton Triggers
 
-![TriggerFlow](https://github.com/tektoncd/triggers/blob/master/images/TriggerFlow.png?raw=true)
+Tekton Triggers is a Tekton component that allows you to detect and extract information from events from a variety of sources and deterministically instantiate
+and execute [`TaskRuns`](https://github.com/tektoncd/pipeline/blob/master/docs/taskruns.md) and [`PipelineRuns`](https://github.com/tektoncd/pipeline/blob/master/docs/pipelineruns.md)
+based on that information. Tekton Triggers can also pass information extracted from events directly to `TaskRuns` and `PipelineRuns`. You install Tekton Triggers on your Kubernetes
+cluster as an extension to Tekton Pipelines.
 
-## Learn More
+## How does Triggers work?
 
-See the following links for more on each of the resources involved:
+Tekton Triggers consists of a controller service that runs on your Kubernetes cluster as well as the following Kubernetes Custom Resource Definitions (CRDs) that extend
+the functionality of Tekton Pipelines to support events:
 
-- [`TriggerTemplate`](triggertemplates.md)
-- [`TriggerBinding`](triggerbindings.md)
-- [`EventListener`](eventlisteners.md)
-- [`ClusterTriggerBinding`](clustertriggerbindings.md)
+*  [`EventListener`](https://github.com/tektoncd/triggers/blob/master/docs/eventlisteners.md) - listens for events at a specified port on your Kubernetes cluster.
+   Specifies one or more `Triggers`.
 
+*  [`Trigger`](https://github.com/tektoncd/triggers/blob/master/docs/triggers.md) - specifies what happens when the `EventListener` detects an event. A `Trigger` specifies
+   a `TriggerTemplate`, a `TriggerBinding`, and optionally, an `Interceptor`.
+
+*  [`TriggerTemplate`](https://github.com/tektoncd/triggers/blob/master/docs/triggertemplates.md) - specifies a template for the `TaskRun` or `PipelineRun` you want to
+   instantiate and execute when EventListener detects an event.
+
+*  [`TriggerBinding`](https://github.com/tektoncd/triggers/blob/master/docs/triggerbindings.md) - specifies the fields in the event payload from which you want to extract
+   data as well as the fields in your `TaskRun` or `PipelineRun` to populate with the extracted values. In other words, it *binds* payload fields to `TaskRun` or
+   `PipelineRun` fields.
+
+*  [`ClusterTriggerBinding`](https://github.com/tektoncd/triggers/blob/master/docs/clustertriggerbindings.md) - a cluster-scoped version of the `TriggerBinding`,
+   especially useful for reuse within your cluster.
+
+*  [`Interceptor`](https://github.com/tektoncd/triggers/blob/master/docs/eventlisteners.md#interceptors) - a "catch-all" event processor for a specific platform that
```

Comment:
I think it's important to mention Interceptors in the overview so that the reader has the complete big picture. We can add a note that Interceptors are being converted to a CRD in PR 960 and then remove it when PR 960 merges?

Reply:
Sounds good to me!